tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-ci-perf-pipeline.yml

```diff
@@ -23,8 +23,9 @@ parameters:
 - name: TrtVersion
   displayName: TensorRT Version
   type: string
-  default: 8.0.1.6
+  default: 8.2.1.8
   values:
+  - 8.2.1.8
```

Comment:
Pipeline file deprecated, change not needed.

Reply:
i deleted the file.